RELEASE_PROCESS.md

```diff
@@ -1,51 +1,85 @@
-# Release Process
+# Release process
 
-This document contains all the necessary information about out release process.
+## Semi-automation
 
-### Merge freeze
+Serverless Framework relies on [semantic commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.4/#summary) which allows to streamline the release process (versioning and changelog generation is automated)
 
-A "merge freeze" will be applied a few days before the release. This means that **NOTHING** will be added to the release branch.
+See proposed [Commit Message Guidelines](https://docs.google.com/document/d/1hKUs3qt_aVp_PBI1UqvfaIqKma3jAJimEoGCRGGbOqs/edit#)
```

Comment:
We might want to move them to another `.md` file later on.

Reply:
I thought about it. Still this document applies to all projects where we practice semantic commits, and maintaining  all copies up to date will be painful. 

If not google docs, then what do you feel will suit it better?

Reply:
Yes, good question. I see the problem of not keeping it DRY...

Maybe we can put it in our template repository and then just link to the the file in such repo. The only concern I have with Google Docs is that they might disappear or the permissions might change. Having the guidelines in VCS feels a little bit more "safe".

Reply:
As discussed, until we agree where to store public dev docs, I'll leave it as it is.

We might want to create a dedicated repository for that, or add some section on our website.

Reply:
Sounds good :+1: